tensorflow/core/util/rocm_solvers.cc

```diff
@@ -219,17 +235,243 @@ void GpuSolver::CheckLapackInfoAndDeleteSolverAsync(
 
 // Macro that specializes a solver method for all 4 standard
 // numeric types.
-#define TF_CALL_LAPACK_TYPES(m) \
+#define TF_CALL_ROCSOLV_TYPES(m) \
```

Comment:
Can we add a table (either in code comments, or in the commit comments) that lists the rocblas/rocsolver/hipsolver routine used to implement a given gpuSolver API ?  Something like

| GpuSolver method | rocblas API | rocsolver API | hipsolver API <br>  (for ROCm 4.5 and greater) |
|:--|:--|:--|:--|
| `Potrf` | | rocsolver_Xportf | hipsolver_Xportrf | 
...
| `Trsm` | rocblas_Xtrsm | | | 
...


Code changes below look ok, but are difficult to extract information from....having the table above will help make it clear, what the code should / is doing

Reply:
See this: 
https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/pull/1530/files#diff-55e07eaefd8ee35563284594ae5ee9db81866c793d4f7105a0c374645ecda4feR16-R34